editoast/src/views/rolling_stocks.rs

```diff
@@ -40,16 +50,77 @@ async fn get_livery(db_pool: Data<DbPool>, path: Path<(i64, i64)>) -> Result<Htt
     }
 }
 
+/// Creation form for a project
+#[derive(Serialize, Deserialize)]
+pub struct RollingStockCreateForm {
+    pub name: String,
+    pub version: String,
+    pub effort_curves: JsonValue,
+    pub base_power_class: String,
+    pub length: f64,
+    pub max_speed: f64,
+    pub startup_time: f64,
+    pub startup_acceleration: f64,
+    pub comfort_acceleration: f64,
+    pub gamma: DieselJson<Gamma>,
+    pub inertia_coefficient: f64,
+    pub features: Vec<String>,
+    pub mass: f64,
+    pub rolling_resistance: DieselJson<RollingResistance>,
+    pub loading_gauge: String,
+    pub metadata: DieselJson<RollingStockMetadata>,
+    pub power_restrictions: Option<JsonValue>,
+}
+
+impl From<RollingStockCreateForm> for RollingStockModel {
+    fn from(rolling_stock: RollingStockCreateForm) -> Self {
+        RollingStockModel {
+            id: None,
```

Comment:
You could implement the Trait Default, and use it.

Reply:
Done